### ISSUE TITLE
Show popular authors/contributors?

### ISSUE THREAD

**Issue by kassemz** [cabrerahector/wordpress-popular-posts#355]
Is it possible to show popular authors/contributors?

**Comment by cabrerahector** [comment]
Hi @kassemz,

Possible? Yes, as long as you're familiar with WordPress development in general (or know someone who is) then yes it's possible.

Is it a feature provided out-of-the-box by WordPress Popular Posts? No, because the ability to assign multiple authors / contributors isn't a core WordPress functionality, it's something usually implemented via third-party plugins (eg. _Co-Authors_ by Publishpress). You'd have to add that information (authors/contributors) to your popular posts programmatically, please see [How can I use my own HTML markup with your plugin?](https://github.com/cabrerahector/wordpress-popular-posts/wiki/5.-FAQ#how-can-i-use-my-own-html-markup-with-your-plugin) for more details.

If you have any other questions please don't hesitate to ask.

**Comment by kassemz** [comment]
Hi Héctor,

Thanks for the quick reply!

I didn't mean multiple authors for the same post. I meant popular authors
based on posts viewed count. So top author would be the one with the most
viewed posts across the site.

In other words, authors with most post views count.

Kassem

**Comment by cabrerahector** [comment]
Yeah, I'm sorry about the confusion, I understood what you meant after re-reading your question and then edited [my previous comment](https://github.com/cabrerahector/wordpress-popular-posts/issues/355#issuecomment-1455222665) accordingly.
